src/main.ts

```diff
@@ -17,13 +17,14 @@ import 'scss/main.scss'
  *
  */
 import {library} from '@fortawesome/fontawesome-svg-core'
-import {faCheckCircle} from '@fortawesome/free-solid-svg-icons'
-import {faCircle, faClone, faTrashAlt} from '@fortawesome/free-regular-svg-icons'
+import {faCheckCircle, faCheck, faMobileAlt} from '@fortawesome/free-solid-svg-icons'
+import {faCircle, faClone, faTrashAlt, faEdit, faEnvelope, faCommentDots} from '@fortawesome/free-regular-svg-icons'
 import {FontAwesomeIcon} from '@fortawesome/vue-fontawesome'
+import CustomIcons from './lib/custom-icons/index'
 import router from './router'
 import App from './App.vue'
 
-library.add(faCircle, faCheckCircle, faClone, faTrashAlt)
+library.add(faCircle, faCheckCircle, faEdit, faCheck, faEnvelope, faMobileAlt, faCommentDots, faClone, faTrashAlt, ...CustomIcons)
```

Comment:
<3

Reply:
@safydy-r  FYI, We can use the icons from font awesome if they are identical to the design(confirmed by Amen). If not, we can add it to the custom icons.